src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: inline-flex;
+  flex-direction: column;
+  max-width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    width: 166px;
+    height: 36px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    height: 14px;
+    line-height: 14px;
+    width: 98px;
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    color: $color-main;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    color: $color-secondary;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main;
+  }
+
+  &__buttom {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 40px;
+    border-radius: 5px;
+    border: 1px solid $color-blue;
+    background-color: $color-blue;
+
+    color: $color-white;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+
+    transition: 0.4s;
```

Comment:
You should write what properties are going to be animated, because now, the browser watching on all properties!